modules/tensor_mechanics/src/materials/ComputeReducedOrderEigenstrain.C

```diff
@@ -57,10 +56,7 @@ ComputeReducedOrderEigenstrain::initQpStatefulProperties()
 void
 ComputeReducedOrderEigenstrain::computeProperties()
 {
-  if (_incremental_form)
-    sumEigenstrainIncremental();
-  else
-    sumEigenstrainTotal();
```

Comment:
I'm glad you can remove this, since I'm trying to get rid of _incremental_form entirely.